templates/shared/repo_search.tmpl

```diff
@@ -65,3 +65,4 @@
 		<span data-tooltip-content="{{ctx.Locale.Tr "explore.relevant_repositories_tooltip"}}">{{ctx.Locale.Tr "explore.relevant_repositories" (printf "?only_show_relevant=0&sort=%s&q=%s&language=%s" $.SortType (QueryEscape $.Keyword) (QueryEscape $.Language))}}</span>
 	</div>
 {{end}}
+<div class="divider"></div>
```

Comment:
Why an extra "divider" here .... maybe it is done to make the UI look better, but maybe it could also not be considered as the purpose of "unifying search boxes"

Reply:
It is present on the other pages like explore/users or explore/orgs. I could have moved it out to a separate PR, but I don't think it's necessary.